docs/user-guide/petset.md

```diff
@@ -0,0 +1,278 @@
+---
+---
+
+* TOC
+{:toc}
+
+__Terminology and conventions__
+
+Throughout this doc you will see a few terms that are sometimes used interchangeably elsewhere, that might cause confusion. This section attempts to clarify them.
+
+* Node: A single virtual or physical machine in a Kubernetes cluster.
+* Cluster: A group of nodes in a single failure domain, unless mentioned otherwise.
+* Persistent Volume Claim (PVC): A request for storage, typically a [persistent volume](/docs/user-guide/persistent-volumes/walkthrough.md).
```

Comment:
Take .md out of the URL and replace with /

Do this throughout the PR, I'm just mentioning it once here.